.github/workflows/hrm-ecs-deploy.yml

```diff
@@ -1,50 +1,23 @@
-# This workflow will build and push a new container image to Amazon ECR,
```

Comment:
These docs were very out of date and I wasn't sure they were worth updating/keeping up to date so I removed them. If others think they are worth updating, I can do that.